app/controllers/swagger_yard/swagger_controller.rb

```diff
@@ -21,7 +21,7 @@ def doc
 
     private
     def resource_listing
-      SwaggerYard::ResourceListing.new(Rails.root+'app/controllers/**/*.rb', Rails.root+'app/models/**/*.rb')
+      SwaggerYard::ResourceListing.all
```

Comment:
Do you think we'll ever want to allow for injecting different paths? In the case of something outside of Rails.